Kosaraju_Algorithm/Kosaraju_algorithm.dart

```diff
@@ -0,0 +1,220 @@
+/**
+ *   Kosaraju's Algorithm
+ * ----------------------------------------
+ * 
+ * This algorithm is used to find the number of strongly connected components
+ * in a directed graph. It uses DFS. It performs DFS twice to achieve the goal based on the fact that
+ * finishing time of starting vertex will be greater than the finishing times of vertices in other SCCs.
+ * 
+ * References - Geeksforgeeks
```

Comment:
try to write explain in your own words

Reply:
I wrote in my own words. I only used geeksforgeeks to understand the algorithm. Since its a good practice to mention the references, I did it.

Reply:
Since, I wrote it already in my own words, please let me know if I have to change it again.

Reply:
If you written in your own language, then the reference is not needed